### PR TITLE
Add links for Ajv i18n and custom errors

### PR DESCRIPTION
**Pull request by dalisoft**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by dalisoft** [comment]
@mcollina Can you review change? Thanks :)

File: docs/Validation-and-Serialization.md [hunk 1]
```diff
@@ -367,6 +367,8 @@ fastify.setErrorHandler(function (error, request, reply) {
 })
 ```
 
+If you want custom error response in schema without headaches and quickly, you can take a look at [here](https://github.com/epoberezkin/ajv-errors)
```
Comment (mcollina) [inline docs/Validation-and-Serialization.md]: Can you also link Ajv i18n?

Comment (dalisoft) [reply docs/Validation-and-Serialization.md]: I linked Ajv i18n [here](https://github.com/fastify/fastify/pull/1588/files#diff-71112d6db09dd080118d76d15c362f4eR503)

File: docs/Validation-and-Serialization.md [hunk 2]
```diff
@@ -367,6 +367,8 @@ fastify.setErrorHandler(function (error, request, reply) {
...
+
```
Comment (Eomm) [inline docs/Validation-and-Serialization.md]:
```suggestion
**Example**
```js
const fastify = require('fastify')({ logger: true })

const Ajv = require('ajv')
const AjvErrors = require('ajv-errors')
const ajv = new Ajv({ allErrors: true, jsonPointers: true })
AjvErrors(ajv)

const schema = {
  $id: 'hello',
  type: 'object',
  required: ['name'],
  properties: {
    name: {
      type: 'string'
    }
  },
  additionalProperties: false,
  errorMessage: 'my custom message'
}

fastify.get('/',
  {
    schema: {
      querystring: schema
    },
    schemaCompiler: schema => {
      const validate = ajv.compile(schema)
      return validate
    }
  }, (req, res) => res.send('hello'))
fastify.listen(3000)
```
```

**Comment by mcollina** [comment]
@dalisoft any updates on this?

**Comment by dalisoft** [comment]
@mcollina No, i think it's enough. I solved my issue with fastify core methods and works as excepted

**Comment by mcollina** [comment]
In https://github.com/fastify/fastify/pull/1588#pullrequestreview-225122882, @Eomm is asking to add an example. Would you mind adding it to this PR?

**Comment by dalisoft** [comment]
I'm not using ajv i18n or custom error plug-in, i'm did it with internally codes/methods, so don't think this related to this PR

**Comment by Eomm** [comment]
The example is how to configure custom error messages.

Anyway I'm not blocking this PR, I will save this example for future "wide-examples-with-fastify" idea that I have in mind 😃

**Comment by dalisoft** [comment]
@mcollina @Eomm Changed a line with [example](https://gist.github.com/dalisoft/a6a088860e724996d0baecae59ace329) of error handling with i18n and custom messages with core methods.

File: docs/Validation-and-Serialization.md [hunk 3]
```diff
@@ -498,3 +500,6 @@ const refToSharedSchemaDefinitions = {
 - [Understanding JSON schema](https://spacetelescope.github.io/understanding-json-schema/)
 - [fast-json-stringify documentation](https://github.com/fastify/fast-json-stringify)
 - [Ajv documentation](https://github.com/epoberezkin/ajv/blob/master/README.md)
+- [Ajv i18n](https://github.com/epoberezkin/ajv-i18n)
+- [Ajv custom errors](https://github.com/epoberezkin/ajv-errors)
+- Custom error handling with core methods with error file dumping [example](https://gist.github.com/dalisoft/a6a088860e724996d0baecae59ace329)
```
Comment (Eomm) [inline docs/Validation-and-Serialization.md]: Sorry, but I don't think it is a nice idea to link an external gist in docs.

I'll try to push the example in the [example repo](https://github.com/fastify/example/)

Comment (dalisoft) [reply docs/Validation-and-Serialization.md]: Ok, thanks :)

Comment (Eomm) [inline docs/Validation-and-Serialization.md]: Finally, we have the right URL 💪
```suggestion
- Custom error handling with core methods with error file dumping [example](https://github.com/fastify/example/tree/master/validation-messages)
```

Comment (dalisoft) [reply docs/Validation-and-Serialization.md]: Thanks @Eomm

**Pull request merged**